apps/src/templates/studioHomepages/TeacherResources.story.jsx

```diff
@@ -1,15 +1,26 @@
 import React from 'react';
 import TeacherResources from './TeacherResources';
+import {Provider} from 'react-redux';
+import {reduxStore} from '@cdo/storybook/decorators';
 
-export default storybook => {
-  return storybook
-    .storiesOf('Homepages/Teachers/TeacherResources', module)
-    .withReduxStore()
-    .addStoryTable([
-      {
-        name: 'Resources for teachers',
-        description: `This is the TeacherResources section that will be used on the teacher homepage.`,
-        story: () => <TeacherResources />
-      }
-    ]);
+export default {
+  title: 'TeacherResources',
+  component: TeacherResources
 };
+
+//
+// TEMPLATE
+//
+
+const Template = args => (
+  <Provider store={reduxStore()}>
+    <TeacherResources {...args} />
```

Comment:
What do we think about deleting this storybook file (if there are unit tests)?

Reply:
Whoops, I definitely got the files mixed up in the PR description. updated that and the jira task to add unit tests for TeacherResources, and delete this storybook file when that's done.